inc/json-schemas/intelligences/biographer-intelligence-1.5.json

```diff
@@ -0,0 +1,51 @@
+{
```

Comment:
New update should remove the necessity to getSummary when it is provided in a message

Reply:
done as in last commit